### PR TITLE
Enabling/Disabling registration

### PR DESCRIPTION
**Pull request by jasonvanmalder**

### CODE REVIEW

File: src/components/administration/AdministrationSettingsPage.js [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+export const AdministrationSettingsPage = () => {
+  const [session] = useState(
+    localStorage.getItem('token')
+      ? jwtDecode(localStorage.getItem('token'))
+      : null
+  )
+
+  const [registrationState, setRegistrationState] = useState(null)
+
+  useEffect(() => {
```
Comment (Sylv11) [inline src/components/administration/AdministrationSettingsPage.js]: Ca le fait en boucle alors ?

Comment (jasonvanmalder) [reply src/components/administration/AdministrationSettingsPage.js]: No, there is an empty array as second parameter (line 74)

Comment (Sylv11) [reply src/components/administration/AdministrationSettingsPage.js]: Oh yes right, it only needs an empty array to stop the loop
10 years in England, right ?

Comment (jasonvanmalder) [reply src/components/administration/AdministrationSettingsPage.js]: Yes, that's right and yes, that's right.

Comment (jasonvanmalder) [reply src/components/administration/AdministrationSettingsPage.js]: NOW APPROVE AND APPROVE BITSKY-SERVER's PR

Comment (Sylv11) [reply src/components/administration/AdministrationSettingsPage.js]: Oh cool
I already approved bitsky-server's PR

Comment (jasonvanmalder) [reply src/components/administration/AdministrationSettingsPage.js]: Ok

**Pull request merged**